Dynatrace/dynatrace-operator
# Add nodeAffinity to csi daemonset

## Description

Ticket: https://dt-rnd.atlassian.net/browse/K8S-9882

In order to align operator components, we should add nodeAffinity to the CSI Deamonset, as Operator/webhook already have it.

## How can this be tested?

Rollout the CSI Driver on a kubernetes cluster -> check if nodeAffinity is set (GKE currently has another bug, as normal GKE is detected as autopilot, will fix that in another PR) so try to not use GKE for testing
